src/main/java/sirius/biz/web/Action.java

```diff
@@ -20,10 +20,10 @@
  */
 public class Action {
 
-    private static final String[] COLOR_CLASSES = {"text-sirius-cyan",
-                                                   "text-sirius-green",
+    private static final String[] COLOR_CLASSES = {"text-sirius-green",
```

Comment:
Commit message says the color palette was extended but no new colors were added

Reply:
Updated would have been a better verb ;-)